multihost_job.py

```diff
@@ -154,6 +159,49 @@ def run_create_resources(startup_script_file):
   captured_output = subprocess.run(command, check=False, shell=True, capture_output=True)
   return captured_output
 
+def run_create_resources_curl(startup_script):
+  """ Run the Create Queued Resources (CQR) request with curl command"""
+  data = {
+        "tpu": {
+            "node_spec": [
+                {
+                    "parent": f"projects/{args.PROJECT}/locations/{args.ZONE}",
+                    "node": {
+                        "accelerator_type": args.TPU_TYPE,
+                        "runtime_version": args.VERSION,
+                        "network_config": {
+                            "network": "default",
+                            "subnetwork": "default",
+                            "enable_external_ips": True
+                        },
+                        "scheduling_config": {
+                            "maintenance_interval": 2
+                        },
+                        "metadata": {
+                            "startup-script": startup_script
+                        }
+                    },
+                    "multi_node_params": {
```

Comment:
Remove multi_node params - this should not be a field in single slice, so we should only add it after first checking num_slices > 1